lib/docs/scrapers/node.rb

```diff
@@ -22,12 +22,17 @@ class Node < UrlScraper
       Node.js is a trademark of Joyent, Inc. and is used with its permission.<br>
       We are not endorsed by or affiliated with Joyent.
     HTML
-
+    
     version do
-      self.release = '19.0.1'
+      self.release = '20.2.0'
       self.base_url = 'https://nodejs.org/api/'
     end
 
+    version '19 LTS' do
```

Comment:
19 isn't an LTS release.

Reply:
Sorry, since version 19 is not an LTS release and its EOL is approaching in about a week, it should not be listed, ig.